<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmsView</class>
 <widget class="QWidget" name="SmsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SMS</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="2">
        <widget class="QGroupBox" name="gbSmsStorageCapacity">
         <property name="title">
          <string>Storage capacity</string>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="labelSmsStorageCapacitySIM">
            <property name="text">
             <string>SIM (used/total):</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="labelSmsStorageCapacitySIMValue">
            <property name="text">
             <string>0/0</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelSmsStorageCapacityModem">
            <property name="text">
             <string>Modem (used/total):</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="labelSmsStorageCapacityPhoneValue">
            <property name="text">
             <string>0/0</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelSmsStorageCapacityArchive">
            <property name="text">
             <string>Local archive:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="labelSmsStorageCapacityArchiveValue">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QToolButton" name="newSmsShowButton">
         <property name="text">
          <string>New SMS</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/sms_create.png</normaloff>icons/sms_create.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QToolButton" name="showFromModem">
           <property name="toolTip">
            <string>Show SMS from modem</string>
           </property>
           <property name="text">
            <string>Show SMS from modem</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/sms_modem.png</normaloff>icons/sms_modem.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="showFromSim">
           <property name="toolTip">
            <string>Show SMS from SIM</string>
           </property>
           <property name="text">
            <string>Show SMS from SIM</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/sms_sim.png</normaloff>icons/sms_sim.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="showUnread">
           <property name="toolTip">
            <string>Show unread SMS</string>
           </property>
           <property name="text">
            <string>Show unread SMS</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/sms_unread.png</normaloff>icons/sms_unread.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="showIncome">
           <property name="toolTip">
            <string>Show income SMS</string>
           </property>
           <property name="text">
            <string>Show income SMS</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/sms_income.png</normaloff>icons/sms_income.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="showDrafts">
           <property name="toolTip">
            <string>Show drafts SMS</string>
           </property>
           <property name="text">
            <string>Show drafts SMS</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/sms_draft.png</normaloff>icons/sms_draft.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="showSent">
           <property name="toolTip">
            <string>Show sent SMS</string>
           </property>
           <property name="text">
            <string>Show sent SMS</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/sms_sent.png</normaloff>icons/sms_sent.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="gbSmsCount">
         <property name="title">
          <string>SMS count</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="labelSmsNewCount">
            <property name="text">
             <string>New:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="labelSmsNewCountValue">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelSmsIncomeCount">
            <property name="text">
             <string>Income:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="labelSmsIncomeCountValue">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelSmsDraftsCount">
            <property name="text">
             <string>Drafts:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="labelSmsDraftsCountValue">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelSmsSentCount">
            <property name="text">
             <string>Sent:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="labelSmsSentCountValue">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="5" column="0" colspan="3">
        <widget class="QPlainTextEdit" name="currentSmsInfo">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QLabel" name="currentSmsInfoLabel">
         <property name="text">
          <string>Selected SMS</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="1" colspan="6">
        <widget class="QTableView" name="smsView">
         <property name="contextMenuPolicy">
          <enum>Qt::ActionsContextMenu</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QToolButton" name="moveToArchiveBtn">
         <property name="text">
          <string>Move to archive</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QToolButton" name="copyToArchiveBtn">
         <property name="text">
          <string>Copy to archive</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="replyBtn">
         <property name="text">
          <string>Reply</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QToolButton" name="deleteBtn">
         <property name="text">
          <string>Delete</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="3">
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="newSmsGroupBox">
     <property name="title">
      <string>Create new SMS</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelReceiver">
        <property name="text">
         <string>Receiver:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="5">
       <widget class="QLineEdit" name="smsReceiver">
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="7">
       <widget class="QLabel" name="labelSmsSymbolsLeft">
        <property name="text">
         <string>Symbols left:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="11">
       <widget class="QLabel" name="smsUsed">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="10">
       <widget class="QLabel" name="labelSmsUsed">
        <property name="text">
         <string>SMS used:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QLabel" name="smsSymbolsLeft">
        <property name="text">
         <string>160</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="14">
       <layout class="QHBoxLayout" name="horizontalLayoutSmsSendText">
        <item>
         <widget class="QPlainTextEdit" name="smsText">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>120</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayoutSmsButtons">
          <item>
           <widget class="QPushButton" name="sendSms">
            <property name="text">
             <string>Send</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="storeSms">
            <property name="text">
             <string>Store to Drafts</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Maximum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="9">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>showFromModem</tabstop>
  <tabstop>showFromSim</tabstop>
  <tabstop>showUnread</tabstop>
  <tabstop>showIncome</tabstop>
  <tabstop>showDrafts</tabstop>
  <tabstop>showSent</tabstop>
  <tabstop>deleteBtn</tabstop>
  <tabstop>replyBtn</tabstop>
  <tabstop>copyToArchiveBtn</tabstop>
  <tabstop>moveToArchiveBtn</tabstop>
  <tabstop>smsView</tabstop>
  <tabstop>newSmsShowButton</tabstop>
  <tabstop>smsReceiver</tabstop>
  <tabstop>smsText</tabstop>
  <tabstop>sendSms</tabstop>
  <tabstop>storeSms</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
